scripts/governance/README.md

```diff
@@ -0,0 +1,82 @@
+# Governance Interaction
+
+## Setup
+
+First install all the npm packages:
+
+```
+npm i
+```
+
+And then install all the python packages using pip3:
+
+```
+pip3 install -r requirements.txt
+```
+
+## Interacting with RSK Testnet
+
+1. Add account with RBTC balance to brownie
+
+```
+brownie accounts new rskdeployer
+```
+
+2. Add network Rsk-testnet
+
+```
+brownie networks add rsk-testnet host=https://testnet.sovryn.app/rpc chainid=31
```

Comment:
```suggestion
brownie networks add rsk testnet host=https://testnet.sovryn.app/rpc chainid=31
```

Entering the original command returned an error:

```
  File "brownie/_cli/__main__.py", line 64, in main
    importlib.import_module(f"brownie._cli.{cmd}").main()
  File "brownie/_cli/networks.py", line 68, in main
    fn(*args["<arguments>"])
  File "brownie/_cli/networks.py", line 134, in _add
    _validate_network(new, PROD_REQUIRED)
  File "brownie/_cli/networks.py", line 316, in _validate_network
    raise ValueError(f"Network is missing required field(s): {', '.join(missing)}")
ValueError: Network is missing required field(s): host
```

Entering my suggested command returned a successful result:

```
SUCCESS: A new network 'testnet' has been added
  └─testnet
    ├─id: testnet
    ├─chainid: 31
    └─host: https://testnet.sovryn.app/rpc
```